Objects/Converters/ConverterRevit/ConverterRevitShared/Partial Classes/ConvertWall.cs

```diff
@@ -205,6 +205,7 @@ public Base WallToSpeckle(DB.Wall revitWall, out List<string> notes)
       }
       else
       {
+        speckleWall.displayValue = new List<Mesh>(); // avoids null value for curtain walls
```

Comment:
did this slip in by mistake?

Reply:
No this was a change on `main` that wasn't merged to `dev` yet, now resolved by https://github.com/specklesystems/speckle-sharp/pull/2968